core/src/main/java/org/elasticsearch/index/translog/TranslogStats.java

```diff
@@ -30,20 +31,29 @@
 
     private long translogSizeInBytes;
     private int numberOfOperations;
+    private long uncommittedSizeInBytes;
+    private int  uncommittedOperations;
```

Comment:
I agree, but I think we need to do this consistently and I didn't want to touch the other `numberOfOperations` field (and the bwc code that entails) in the same PR. That's why I did it like this.